internal/controller/kustomization_controller.go

```diff
@@ -1160,6 +1180,24 @@ func (r *KustomizationReconciler) patch(ctx context.Context,
 	return nil
 }
 
+func (r *KustomizationReconciler) getClientAndPoller(
```

Comment:
Please add a comment to this function, explain what it does

Reply:
Fixed, thanks!